oak-segment-azure/src/main/java/org/apache/jackrabbit/oak/segment/azure/tool/AzureCheck.java

```diff
@@ -18,8 +18,8 @@
 
 import com.google.common.io.Files;
 import com.microsoft.azure.storage.blob.CloudBlobDirectory;
-import org.apache.jackrabbit.oak.segment.azure.AzurePersistence;
-import org.apache.jackrabbit.oak.segment.azure.AzureStorageCredentialManager;
+import org.apache.jackrabbit.oak.segment.azure.v8.AzurePersistenceV8;
```

Comment:
Do we have this class also upgraded to Azure SDK v12 somewhere?

Reply:
No, all "Tooling" (compaction, consistency check...) will be done in a different PR.